plugins/chat/plugin.rb

```diff
@@ -421,7 +421,6 @@ def self.onebox_template
       following: false,
       limit: 1,
       status: :open,
```

Comment:
This should also still exclude the DM channels

Reply:
There is no need to pass the option in anymore because `Chat::ChannelFetcher.secured_public_channel_search` will always set `exclude_dm_channels` to `true`.

Reply:
Oh right of course, thanks for fixing the other one